Sources/NIOPosix/SelectableEventLoop.swift

```diff
@@ -77,10 +77,12 @@ internal final class SelectableEventLoop: EventLoop {
     internal var scheduledTaskCounter = ManagedAtomic<UInt64>(0)
     @usableFromInline
     internal var _scheduledTasks = PriorityQueue<ScheduledTask>()
+    @usableFromInline
+    internal var _immediateTasks = CircularBuffer<UnderlyingTask>()
 
     // We only need the ScheduledTask's task closure. However, an `Array<() -> Void>` allocates
```

Comment:
nit: remove the comment, no longer relevant as you're now holding `UnderlyingTask` anyway

Reply:
done.